public/video-ui/src/util/parseGridMetadata.js

```diff
@@ -10,22 +10,69 @@ function parseMimeType(mimeType) {
   return mimeType;
 }
 
-function parseAsset(asset) {
+function parseAsset(asset, aspectRatio) {
   return {
     file: asset.secureUrl,
     mimeType: parseMimeType(asset.mimeType),
     size: asset.size,
+    aspectRatio: aspectRatio,
     dimensions: {
       width: asset.dimensions.width,
-      height: asset.dimensions.height
+      height: asset.dimensions.height,
```

Comment:
Probably since the build passed but I seemed to have removed that anyway on my branch without pushing up the changes...